dali/test/python/test_pipeline.py

```diff
@@ -1118,30 +1143,22 @@ def define_graph(self):
     pipe.build()
     pipe.run()
     for method in [pipe.schedule_run, pipe.share_outputs, pipe.release_outputs, pipe.outputs]:
-        try:
+        with assert_raises(RuntimeError):
             method()
-            assert(False)
-        except RuntimeError:
-            assert(True)
     # disable check
     pipe.enable_api_check(False)
     for method in [pipe.schedule_run, pipe.share_outputs, pipe.release_outputs, pipe.outputs]:
-        try:
-            method()
-            assert(True)
-        except RuntimeError:
-            assert(False)
+        method()
```

Comment:
missing `assert_raises`?

Reply:
Note that it asserts False in case of the exception, I think this one was meant to simply pass.